### PR TITLE
added pagination plugin

### LINKED ISSUE #521: [NG] Add pagination plugin
**Desired Behavior**
Add and configure [gatsby plugin for pagination](https://www.gatsbyjs.org/packages/gatsby-awesome-pagination/) in[ layer5-ng](https://github.com/layer5io/layer5/tree/layer5-ng). Related to #481

### PR DESCRIPTION
**Pull request by codetheorem**
Signed-off-by: Hrishikesh Agarwal <hrishikeshagarwalv@gmail.com>

**Description**

Added pagination plugin according to this https://www.gatsbyjs.com/plugins/gatsby-awesome-pagination/.

This PR fixes #521 

**Notes for Reviewers**
It requires configuration.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@codetheorem this is exciting. Is the site building locally for you? Build logs - https://app.netlify.com/sites/layer5ng/deploys/5f64fd772868170007d180d0

**Comment by codetheorem** [comment]
@leecalcote there is some problem I think I will commit it after resolving the error.

**Comment by Tanuj22** [comment]
> @leecalcote there is some problem I think I will commit it after resolving the error.

Nice. Let us know if you need any help debugging it.

**Comment by leecalcote** [comment]
@codetheorem would it help to troubleshoot this together?

**Comment by codetheorem** [comment]
@leecalcote thank you for your help I will let you know If I will need any help.

File: gatsby-node.js [hunk 1]
```diff
@@ -71,6 +72,14 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
       }
   `);
 
+  paginate({
```
Comment (Tanuj22) [inline gatsby-node.js:75]: Shouldn't this we added after line 93 as `blogs` is define there

**Comment by leecalcote** [comment]
We need to advance here. @codetheorem do you think you will get back to the PR today? If not, that’s no problem. Another community member can take this one forward, With any number of other ones open for your consideration.

**Pull request closed without merging**